### PR TITLE
exercises: Add a brief introducion

### PR DESCRIPTION
**Pull request by core-man**
Addresses #205.

### CODE REVIEW

File: source/exercises/introduction.rst [hunk 1]
```diff
@@ -3,7 +3,33 @@
...
+目的是帮助地震学新手快速熟悉地震学数据下载、处理以及分析的基础代码，然后直接可以开展实际科研工作。
+
+地震台站记录到的地震图会实时或隔一段时间后传输至数据中心（\ **Data Center**\ ）。一般
+情况下，地震学科研工作者直接可以从数据中心下载到公开的波形数据（time series data）。
+此外，一些数据中心也会提供地震目录（catalog）和台站元数据（medadata）。
+目前全球比较知名的数据中心有 `Incorporated Research Institutions for Seismology (IRIS) <https://www.iris.edu/hq/>`__\ 、
+`Observatories & Research Facilities for European Seismology (ORFEUS) <http://www.orfeus-eu.org/>`__\ 、
+`Hi-net <https://hinetwww11.bosai.go.jp/auth/?LANG=en>`__ 等。
+
+.. figure:: workflow.jpg
```
Comment (seisman) [inline source/exercises/introduction.rst:21]: Quick comment on the figure: 

Do you want to try https://build-me-the-docs-please.readthedocs.io/en/latest/Using_Sphinx/UsingGraphicsAndDiagramsInSphinx.html?

Comment (core-man) [reply source/exercises/introduction.rst:21]: Not now. instead we will use it later

File: source/exercises/introduction.rst [hunk 2]
```diff
@@ -3,7 +3,33 @@
 
 .. note::
 
-    地震学实践部分正在全力编写中，计划的实践列表请查看
+    地震学实践的各章节正在全力编写中，计划的实践列表请查看
     `seismology101 issue #205 <https://github.com/seismo-learn/seismology101/issues/205>`__
 
-这一部分介绍日常地震学科研工作中的常见实践。
+日常地震学科研工作中，使用地震学数据开展研究至前，往往需要先获取观测资料，并做预处理和分析。
```
Comment (seisman) [inline source/exercises/introduction.rst:9]:
```suggestion
日常地震学科研工作中，使用地震学数据开展研究前，往往需要先获取观测资料，并做预处理和分析。
```

File: source/exercises/introduction.rst [hunk 3]
```diff
@@ -3,7 +3,33 @@
...
+经处理和分析得到可靠数据可用于研究震源、地震学成像以及探索地球深部等。因此，这一部分将介绍
+日常地震学科研工作中的常见实践，主要以实际工作中会用到的代码为主。
```
Comment (seisman) [inline source/exercises/introduction.rst:11]:
```suggestion
日常地震学科研工作中的常用工作流程，主要以实际工作中会用到的代码为主。
```

File: source/exercises/introduction.rst [hunk 4]
```diff
@@ -3,7 +3,33 @@
...
+目的是帮助地震学新手快速熟悉地震学数据下载、处理以及分析的基础代码，然后直接可以开展实际科研工作。
+
+地震台站记录到的地震图会实时或隔一段时间后传输至数据中心（\ **Data Center**\ ）。一般
+情况下，地震学科研工作者直接可以从数据中心下载到公开的波形数据（time series data）。
+此外，一些数据中心也会提供地震目录（catalog）和台站元数据（medadata）。
+目前全球比较知名的数据中心有 `Incorporated Research Institutions for Seismology (IRIS) <https://www.iris.edu/hq/>`__\ 、
+`Observatories & Research Facilities for European Seismology (ORFEUS) <http://www.orfeus-eu.org/>`__\ 、
+`Hi-net <https://hinetwww11.bosai.go.jp/auth/?LANG=en>`__ 等。
```
Comment (seisman) [inline source/exercises/introduction.rst:19]: Perhaps we should only mention IRIS DMC here?

Comment (core-man) [reply source/exercises/introduction.rst]: done

**Review by seisman — APPROVED** [review]

File: source/exercises/introduction.rst [hunk 5]
```diff
@@ -3,7 +3,31 @@
 
 .. note::
 
-    地震学实践部分正在全力编写中，计划的实践列表请查看
+    地震学实践的各章节正在全力编写中，计划的实践列表请查看
     `seismology101 issue #205 <https://github.com/seismo-learn/seismology101/issues/205>`__
 
-这一部分介绍日常地震学科研工作中的常见实践。
+日常地震学科研工作中，使用地震学数据开展研究前，往往需要先获取观测资料，并做预处理和分析。
+经处理和分析得到可靠数据可用于研究震源、地震学成像以及探索地球深部等。因此，这一部分将介绍
```
Comment (core-man) [inline source/exercises/introduction.rst:10]:
```suggestion
经处理和分析得到的可靠数据可用于研究震源、地震学成像以及探索地球深部等。因此，这一部分将介绍
```

**Pull request merged**